k8s/migratev2/migrator.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:alpine as base
+RUN apk add --no-cache --quiet \
+      musl-dev gcc ca-certificates mailcap \
+      upx tzdata zip openssh git bash zsh tcsh && \
+    update-ca-certificates
+
+FROM base as builder
+ARG version
+ARG commit
+ARG dirty
+
+ENV VERSION ${version}
+ENV GIT_COMMIT ${commit}
+ENV GIT_DIRTY ${dirty}
+
+ENV GO111MODULE on
+ENV GOBIN ${GOPATH}/bin
+RUN mkdir -p ${GOBIN} && mkdir -p /stage/usr/bin
+
+WORKDIR ${GOPATH}/src/github.com/oneconcern
+RUN git clone https://github.com/oneconcern/datamon datamon1 && cd datamon1 && git checkout v1.0.0
```

Comment:
Download the release?

Reply:
yes could do that. I was not so sure about the artifact... In doubt I've built from source.
Anyway, this is done once: I pushed the image on gcr and it probably won't be updated any time soon